www/src/pages/table/forms/TransformationForm.tsx

```diff
@@ -0,0 +1,278 @@
+          <Col span={10}>
+            <Form.Item
+              name="datapath"
+              label="datapath"
+              rules={[{ required: true }]}
+            >
+              <Select
+                mode="tags"
+                allowClear={true}
+                style={{
+                  width: "100%",
+                }}
+                placeholder="Please select columns"
+                options={table.columns.map((column) => ({
+                  label: column,
+                  value: column,
+                }))}
+              />
+            </Form.Item>
+          </Col>
+          <Col span={10}>
+            <Form.Item
+              name="outputpath"
+              label="outputpath"
+              rules={[{ required: true }]}
+            >
+              <Select
+                mode="tags"
+                allowClear={true}
+                style={{
+                  width: "100%",
+                }}
+                options={table.columns.map((column) => ({
+                  label: column,
+                  value: column,
+                }))}
+                placeholder="Please select columns"
+              />
+            </Form.Item>
+          </Col>
+        </Row>
+        <Row justify="center" align="top">
+          <Col span={24}>
+            <Form.Item
+              name="code"
+              label="Language: Restricted Python"
+              labelCol={{ span: 24 }}
+              style={{ fontWeight: 500, padding: 0 }}
+            >
+              <CustomEditor />
+            </Form.Item>
+          </Col>
+        </Row>
+        <Row gutter={[8, 8]} justify="space-between">
+          <Col span={2} flex="auto">
+            <Form.Item>
+              <Button
+                style={{ width: "100%" }}
+                type="primary"
+                onClick={onExecute}
+              >
+                Execute
+              </Button>
+            </Form.Item>
+          </Col>
+          <Col span={2} flex="auto">
+            <Form.Item>
+              <Button style={{ width: "100%" }} type="primary">
+                Save
+              </Button>
+            </Form.Item>
+          </Col>
+          <Col span={2} flex="auto">
+            <Form.Item>
+              <Button style={{ width: "100%" }} type="primary">
+                Reset
+              </Button>
+            </Form.Item>
+          </Col>
+          <Col span={11} style={{ paddingLeft: "2rem" }}>
+            <Form.Item
+              name="onerror"
+              label="on error"
+              rules={[{ required: true }]}
+              initialValue={4}
+            >
+              <Radio.Group>
+                <Radio value={1}>Set To Blank</Radio>
+                <Radio value={2}>Store Error</Radio>
+                <Radio value={3}>Keep Original</Radio>
+                <Radio value={4}>Abort</Radio>
+              </Radio.Group>
+            </Form.Item>
+          </Col>
+          <Space></Space>
```

Comment:
you said you removed this, but it's still here.

Reply:
removed it.